api/proquest/client.py

```diff
@@ -74,6 +76,26 @@ class ProQuestAPIClientConfiguration(ConfigurationGrouping):
         required=False,
     )
 
+    stop_on_empty_page = ConfigurationMetadata(
```

Comment:
What cases would we ever want to not set this setting? Do we need this as a configuration setting? This functionality seems like the kind of thing that our implementation team doesn't need to know about. Adding this is a setting will just be confusing to them.

Reply:
This functionality is a workaround that we might want to get rid of in the future, if ProQuest fixes the pagination issue.

Reply:
Lets remove the config setting, and have this be the default behavior. If we come across blank pages in the middle of the feed, we can reevaluate.

Reply:
I removed the configuration setting.